lib/event_store_http_client/events/write.rb

```diff
@@ -14,14 +14,12 @@ class Write
         dependency :settings
         dependency :client
 
-        def initialize(type, data, stream_name, version)
```

Comment:
That seems right. I guess we can see how that shakes out when we write more of the concrete code.